relayer/pkg/starknet/starknet.go

```diff
@@ -0,0 +1,70 @@
+package starknet
```

Comment:
This source file should be moved to `ops` module (currently under `relayer/ops`) to a `devnet` package, as this is the context of this file.

Reply:
Moved and renamed